### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by biruk-bereka**
Add Work Section and About me to Mobile Version
         1. Add Work section 
               - Add a headline for the work section
               - Add projects in the work section
         2. Add About me section
               - Add a headline for the about me section
               - Add professions on the about me section

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -109,4 +113,252 @@ body {
...
+    font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #091e42;
+  }
+
+  .project1-description {
+    width: 327px;
```
Comment (sja-thedude) [inline index.css:157]: - [ ] Kindy use display: grid for the main container of the projects in the works section as per the requirement.

File: index.css [hunk 2]
```diff
@@ -109,4 +113,252 @@ body {
...
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    padding: 12px;
+    border: none;
+    color: #fff;
+    background: #ff6b00;
+  }
```
Comment (sja-thedude) [inline index.css:199]: - [ ] Kindly add button interaction for all the buttons in your portfolio as per the Figma design given. Implement the button styling for Active, Hover and Pressed.

Screenshot 2022-04-20 at 10 10 00 PM

File: index.css [hunk 3]
```diff
@@ -109,4 +113,252 @@ body {
...
+
+  .project1-title {
+    margin-bottom: 0;
+    margin-top: 25px;
+    font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
```
Comment (sja-thedude) [inline index.css:152]: - [ ] Kindly make sure that the margin of `work section heading` and `multi-post stories card` on both sides are equal. Fix the margin and make sure that the right side margin has the same margin as on the left side of the card as per the Figma design.

Your design:

https://user-images.githubusercontent.com/70269507/164308370-94a59526-b6a7-4c2e-a6dc-b856c75cf113.mov

Figma design:

Screenshot 2022-04-20 at 10 19 27 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @biruk-bereka, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct Gitflow
✔️ Professional Readme
✔️ Good PR Title and Description

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @biruk-bereka 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**